### ISSUE TITLE
Don't work rec video: Error in video recording

### ISSUE THREAD

**Issue by slacknk** [visualboyadvance-m/visualboyadvance-m#124]
OS-Distro: slackware64-14.2 
Version visualboyadvance-m (wx): 20170603_throttle (from git origin/master)

Build keys: 
```
  SLKCFLAGS="-O2 -fPIC"
  LIBDIRSUFFIX="64"
  ASM=OFF
```
```
 -DCMAKE_C_FLAGS:STRING="$SLKCFLAGS" \
 -DCMAKE_CXX_FLAGS:STRING="$SLKCFLAGS" \
 \
 -DENABLE_ASM_CORE=${ASM} \
 -DENABLE_ASM_SCALERS=${ASM} \
 \
 -DENABLE_WX=ON \
 -DwxWidgets_CONFIG_EXECUTABLE=/usr/lib$LIBDIRSUFFIX/wx/config/gtk2-unicode-3.0 \
 -DwxWidgets_USE_DEBUG=OFF \
 \
 -DENABLE_FFMPEG=ON \
 -DENABLE_SDL=ON \
 -DENABLE_NLS=ON \
 -DENABLE_CAIRO=ON \
 -DENABLE_OPENAL=ON
```

Open anybody ROM, Menu: File - Record - Start video recording...
folder, example: /tmp - filename without spaces - x264/mkv/ogv/avi - save 
and opened window with error:

> Error in video recording (error writing to output file); aborting

Log from terminal:
```
$ visualboyadvance-m 
[libmp3lame @ 0x3cfd400] Channel layout not specified
[avi @ 0x3d00e00] Using AVStream.codec.time_base as a timebase hint to the muxer is deprecated. Set AVStream.time_base instead.
[avi @ 0x3d00e00] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
[avi @ 0x3d00e00] Using AVStream.codec.time_base as a timebase hint to the muxer is deprecated. Set AVStream.time_base instead.
[avi @ 0x3d00e00] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
[swscaler @ 0x3d0a400] Warning: data is not aligned! This can lead to a speedloss
[avi @ 0x3d00e00] Timestamps are unset in a packet for stream 0. This is deprecated and will stop working in the future. Fix your code to set the timestamps properly
[avi @ 0x3d00e00] Encoder did not produce proper pts, making some up.
[avi @ 0x3d00e00] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 1 >= 0
[libmp3lame @ 0x3cfd400] 2 frames left in the queue on closing
```
Installed:
- ffmpeg-3.3.2-x86_64-1alien
- lame-3.99.5-x86_64-1_SBo 
- twolame-0.3.13-x86_64-1_SBo
- x264-20170225-x86_64-1_SBo
- x265-2.4-x86_64-1_SBo

Tried on mint64-17 (ubuntu-14.04) with package 2.0.0~beta3-r201705121847-535a2d5-19 from
https://code.launchpad.net/~sergio-br2/+archive/ubuntu/vbam-trunk/
error with rec too

**Comment by rkitover** [comment]
Can confirm video recording is broken, tried on ubuntu and mac.

**Comment by SnorlaxYum** [comment]
> Can confirm video recording is broken, tried on ubuntu and mac.

Encountered the same error, how to fix it?

**Comment by ZachBacon** [comment]
> > Can confirm video recording is broken, tried on ubuntu and mac.
> 
> Encountered the same error, how to fix it?

You wait till we provide a fix. Currently it's not a high priority bug and if you need to record something, in the interim I suggest obs studio

**Comment by denisfa** [comment]
I am still working on this atm, but development is slow. I hope I finish this soon.

**Comment by rkitover** [comment]
Fixed in 5848feae.
